### PR TITLE
Fix polygon overflow

### LINKED ISSUE #612: Overflow on polygon with 256 sides
Setting the sides to 256 in the shape tool and then drawing causes an overflow panic (assuming you are running with debug asserts). This can be fixed by either casing to a larger integer (i.e. u32) or by subtracting 1 rather than adding one (giving the same result)

### PR DESCRIPTION
**Pull request by Androxium**
Closes #612 

Previously, setting the number of sides in the polygon to 256 (and the drawing) causes an overflow panic. To resolve this issue, we change the polygon side type to 'u32'. Also, we increased the max polygon side value.

Work done
- change the polygon side type to `u32`
- set the max polygon side value to 1000

### CODE REVIEW

File: graphene/src/operation.rs [hunk 1]
```diff
@@ -215,8 +215,8 @@ impl Operation {
 	/// # Safety
 	/// This function reads from uninitialized memory!!!
 	/// Only use if you know what you are doing
-	unsafe fn as_slice(&self) -> &[u8] {
-		core::slice::from_raw_parts(self as *const Operation as *const u8, std::mem::size_of::<Operation>())
+	unsafe fn as_slice(&self) -> &[u32] {
+		core::slice::from_raw_parts(self as *const Operation as *const u32, std::mem::size_of::<Operation>())
```
Comment (0HyperCube) [inline graphene/src/operation.rs:219]: Why are we changing this?

**Review by 0HyperCube — APPROVED** [review]
Looks good

File: graphene/src/operation.rs (hunk 1 above)
Comment (Androxium) [reply graphene/src/operation.rs]: Just tested it again after reverting this change and feature still works as intened, committing the revert

**Review by 0HyperCube — APPROVED** [review]
Looks good.

**Pull request merged**